tests/bot/exts/moderation/test_silence.py

```diff
@@ -405,13 +610,22 @@ async def test_sent_correct_message(self):
             (False, silence.MSG_UNSILENCE_MANUAL, PermissionOverwrite(send_messages=False)),
             (False, silence.MSG_UNSILENCE_MANUAL, PermissionOverwrite(add_reactions=False)),
         )
+
         for was_unsilenced, message, overwrite in test_cases:
             ctx = MockContext()
-            with self.subTest(was_unsilenced=was_unsilenced, message=message, overwrite=overwrite):
+
+            for target in [None, MockTextChannel()]:
+                ctx.channel.overwrites_for.return_value = overwrite
+                if target:
+                    target.overwrites_for.return_value = overwrite
```

Comment:
Setting overrwrites for the target is unnecessary.

Reply:
When the target channel is set, the unsilence wrapper uses its permissions to determine behavior. Without setting the overwrites, the error message changes from `MSG_UNSILENCE_FAIL` to `MSG_UNSILENCE_MANUAL`.

Reply:
Looks like you are correct. You're talking about this, right? https://github.com/python-discord/bot/pull/1297/files#diff-f64697ccedd0a53137e360dba5f9f39c7f7a7b2fa76e29153024d1eb7a1d51c0R228

Reply:
Yeah that's the one. The if statement right below it switches the message.